roles/zabbix_agent/vars/agent2_vars.yml

```diff
@@ -3,7 +3,8 @@ _logfile: /var/log/zabbix/zabbix_agent2.log
 _include: /etc/zabbix/zabbix_agent2.d
 _tls_subject: "{{ zabbix_agent_tlsservercertsubject | default(omit) }}" # FIXME this is not correct and should be removed with 2.0.0, here only to prevent regression
 _win_package: zabbix_agent2-{{ zabbix_version_long }}-windows-amd64-openssl-static.zip
-_win_download_link: "{{ zabbix_win_download_url }}/{{ zabbix_version_long | regex_search('^\\d+\\.\\d+') }}/{{ zabbix_version_long }}/{{ zabbix2_win_package }}"
-_win_logfile: "{{ zabbix_win_install_dir }}\\zabbix_agent2.log"
+_win_download_link: "{{ zabbix_win_download_url }}/{{ zabbix_version_long | regex_search('^\\d+\\.\\d+') }}/{{ zabbix_version_long }}/{{ zabbix2_win_package | default('_win_package') }}"
```

Comment:
See the latest changes.  Support for `zabbix2_win_package` has been removed with the release of 3.0.0

Reply:
What do you suggest as an alternative?

The code below uses the variable `_win_download_link`  when the variables `zabbix_agent_win_download_link`  and `zabbix_agent2_win_download_link` are **not** set.

https://github.com/ansible-collections/community.zabbix/blob/45ed88b2abbaffb8869bcb102ac2ce5ed6694769/roles/zabbix_agent/tasks/Windows.yml#L1-L4

On the other hand, the code below uses the variable `zabbix2_win_package ` that is not set by default.

https://github.com/ansible-collections/community.zabbix/blob/45ed88b2abbaffb8869bcb102ac2ce5ed6694769/roles/zabbix_agent/vars/agent2_vars.yml#L6

Therefore, the script will fail if none of these variables are set when the Role is applied.

Reply:
Maybe I misunderstood your comment. Do you just want to unify the parameter for Zabbix_agent and Zabbix_agent2?